docs/napari-plugin-guide.md

```diff
@@ -160,44 +93,43 @@ User specifies the directory in which the images will be saved. Needed if the *S
 
 Ex. for a 20 um thick cell, the user would first focus in the middle of the cell and then set the following parameters: 
 
-​									`Z Start: 	-12` 
-
-​									 `Z End:	 12`
-
-​									`Z Step:	 0.25` 
+​* `Z Start = -12`
+​* `Z End: 12`
+* `Z Step: 0.25` 
```

Comment:
Line breaks here may not [render](https://github.com/mehta-lab/recOrder/blob/87efe8d693c19b7709cedf97dfb0201b3552552d/docs/napari-plugin-guide.md) as intended

Reply:
Thanks!